src/main.js

```diff
@@ -64,10 +64,13 @@ const ALLACTIONS = [
 // **************************************************************************
 async function main() {
   try {
-    const owner = github.context.repo.owner;
-    const repo = github.context.repo.repo;
+    const ctx = github.context;
+    const { owner, repo } = ctx.repo;
 
-    const issueNumber = core.getInput('issue-number');
+    let defaultNo;
+    if (ctx.eventName === 'issues') defaultNo = ctx.payload.issue.number;
+
+    const issueNumber = core.getInput('issue-number') || defaultNo;
```

Comment:
Maybe we could use optional chaining here? (Not sure if it's supported in github actions)

```ts
const issueNumber = core.getInput('issue-number') || ctx.payload?.issue?.number;
```

Reply:
node 12 does not support